doc/jcli/transaction.md

```diff
@@ -28,3 +28,140 @@ content information of a transaction:
 * `info`
 * `id` to get the **Transaction ID** of the transaction
 * `to-message` to get the hexadecimal encoded message, ready to send with `cli rest message`
+
+
+# Examples
+
+Let's say we have the following utxo
+
+```plaintext
+in_idx: 0
+in_txid: 55762218e5737603e6d27d36c8aacf8fcd16406e820361a8ac65c7dc663f6d1c
+out_addr: ta1sn9u0nxmnfg7af8ycuygx57p5xgzmnmgtaeer9xun7hly6mlgt3pjp0h0rrq782p7z4auve0s4l72egyc88f49mzm8lwuaw7xnw52udky79t3s
+out_value: 100
+```
+
+And we want to transfer 50 lovelaces to the following address
+
+**ta1ssnr5pvt9e5p009strshxndrsx5etcentslp2rwj6csm8sfk24a2wu27hujgyzl4mlc736jaztykud5tqrxw5gd6esmw7rx5zujdzw3ahcaskk**
+
+***
+
+**Note**: 
+
+The following examples assume that you have something like this in your node configuration file (and that you have a locally running node).
+
+You can check the [rest section](./rest.md) for more information on the rest commands.
+
+```yaml
+rest:
+  listen: "127.0.0.1:8443"
+  prefix: "api"
+```
+
+For example, for getting all the utxos:
+`jcli rest v0 utxo get --host http://127.0.0.1:8443/api`
```

Comment:
This command is not going to stay. We have it as a debug end point. but it will be removed as soon as we have a wallet or an explorer to provide this kind of data.